src/core/evaluator.js

```diff
@@ -1176,10 +1176,11 @@ class PartialEvaluator {
     let fontRef;
     if (font) {
       // Loading by ref.
-      if (!(font instanceof Ref)) {
-        throw new FormatError('The "font" object should be a reference.');
+      if (font instanceof Ref) {
+        fontRef = font;
+      } else {
+        warn('The "font" object should be a reference.');
```

Comment:
Is this warning useful ?

Reply:
Probably not, since there's no known issue/bug that's hitting it and given that we print a warning in the fallback code-path below.